add option to save the rois

jburel opened the pull request:
Add a new version to the idr0062 notebook
The notebook shows
- how to load the data from an OMERO.server
- how to segment the data using StarDist
- how to convert the StarDist labels into OME rois.
- how to save the rois back to the server

jburel committed:
add option to save the rois

jburel committed:
remove geojson section

pwalczysko commented on notebooks/idr0062_prediction_save.ipynb:
```diff
@@ -0,0 +1,455 @@
+    "\n",
+    "The notebook shows:\n",
+    "\n",
+    " - how to load image from an OMERO server.\n",
```
```suggestion
    " - How to load an image from an OMERO server.\n",
```

pwalczysko commented on notebooks/idr0062_prediction_save.ipynb:
```diff
@@ -0,0 +1,455 @@
+    "The notebook shows:\n",
+    "\n",
+    " - how to load image from an OMERO server.\n",
+    " - how to segment using a DeepLearning software, [StarDist](https://github.com/stardist/stardist).\n",
```
```suggestion
    " - How to segment using a Deep Learning software, [StarDist](https://github.com/stardist/stardist).\n",
```

pwalczysko commented on notebooks/idr0062_prediction_save.ipynb:
```diff
@@ -0,0 +1,455 @@
+    "\n",
+    " - how to load image from an OMERO server.\n",
+    " - how to segment using a DeepLearning software, [StarDist](https://github.com/stardist/stardist).\n",
+    " - how to convert the stardist labels into OME rois.\n",
```
```suggestion
    " - How to convert the stardist labels into OME ROIs.\n",
```

pwalczysko commented on notebooks/idr0062_prediction_save.ipynb:
```diff
@@ -0,0 +1,455 @@
+    " - how to load image from an OMERO server.\n",
+    " - how to segment using a DeepLearning software, [StarDist](https://github.com/stardist/stardist).\n",
+    " - how to convert the stardist labels into OME rois.\n",
+    " - how to save them back to the OMERO.server.\n",
```
```suggestion
    " - How to save the ROIs back to the OMERO.server.\n",
```

pwalczysko commented on notebooks/idr0062_prediction_save.ipynb:
```diff
@@ -0,0 +1,455 @@
+   "id": "silent-webmaster",
+   "metadata": {},
+   "source": [
+    "# Load Image from an OMERO.server, segment using StarDist and save the results back to the server.\n",
```
```suggestion
    "# Load an Image from an OMERO.server, segment it using StarDist and save the results back to the server.\n",
```

pwalczysko commented on notebooks/idr0062_prediction_save.ipynb:
```diff
@@ -0,0 +1,455 @@
+    "The image is referenced in the paper \"NesSys: a novel method for accurate nuclear segmentation in 3D\" published August 2019 in PLOS Biology: https://doi.org/10.1371/journal.pbio.3000388 and can be viewed online in the [Image Data Resource](https://idr.openmicroscopy.org/webclient/?show=image-6001247).\n",
+    "\n",
+    "\n",
+    "In this notebook, the image is loaded and analyzed using a DeepLearning model software [StarDist](https://github.com/stardist/stardist). The StarDist analysis produces a segmentation, which is then converted and saved to the OMERO.server.\n",
```
```suggestion
    "In this notebook, the image is loaded and analyzed using a Deep Learning software [StarDist](https://github.com/stardist/stardist). The StarDist analysis produces a segmentation, which is then converted and saved to the OMERO.server.\n",
```

pwalczysko commented:
All works fine, except the RoIs have by default quite a thick yellow stroke and a pink fill, which is surprising.
![Screenshot 2022-06-13 at 14 49 39](https://user-images.githubusercontent.com/2478303/173370234-4e3cd395-9a2e-43ec-a449-416c1243ccb2.png)
![Screenshot 2022-06-13 at 14 49 34](https://user-images.githubusercontent.com/2478303/173370239-8e2f5597-df7d-4361-93c7-4ece3123ef3a.png)

jburel committed:
Update notebooks/idr0062_prediction_save.ipynb

Co-authored-by: pwalczysko <p.walczysko@dundee.ac.uk>

jburel committed:
Update notebooks/idr0062_prediction_save.ipynb

Co-authored-by: pwalczysko <p.walczysko@dundee.ac.uk>

jburel committed:
Update notebooks/idr0062_prediction_save.ipynb

Co-authored-by: pwalczysko <p.walczysko@dundee.ac.uk>

jburel committed:
Update notebooks/idr0062_prediction_save.ipynb

Co-authored-by: pwalczysko <p.walczysko@dundee.ac.uk>

jburel commented:
We can change that. I comment that out and use the default

jburel committed:
remove color filling to simplify notebook

jburel committed:
view the generated stardist labels

jburel committed:
add python matching notebook

jburel committed:
Update notebooks/idr0062_prediction_save.ipynb

Co-authored-by: pwalczysko <p.walczysko@dundee.ac.uk>

jburel commented:
@pwalczysko
 I have:
*  fixed the size/color of the shapes
* added viewing of the labels generated by stardist
* added python script matching notebook

jburel committed:
Update notebooks/idr0062_prediction_save.ipynb

Co-authored-by: pwalczysko <p.walczysko@dundee.ac.uk>

pwalczysko commented:
Thank you
